Optical Society of America.js

```diff
@@ -91,21 +97,37 @@ function scrape(doc, url) {
 	var translator = Zotero.loadTranslator('web');
 	// Embedded Metadata
 	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
-	//translator.setDocument(doc);
+	// translator.setDocument(doc);
 
 	translator.setHandler('itemDone', function (obj, item) {
-		if (item.abstractNote ) {
+		item.title = decodeEntities(item.title, doc);
+		item.bookTitle = decodeEntities(item.bookTitle, doc);
+		item.publicationTitle = decodeEntities(item.publicationTitle, doc);
+		item.rights = decodeEntities(item.rights, doc);
+		
+		if (item.abstractNote) {
 			item.abstractNote = ZU.trimInternal(item.abstractNote);
 		}
 
 		item.complete();
 	});
 
-	translator.getTranslatorObject(function(trans) {
+	translator.getTranslatorObject(function (trans) {
 		trans.doWeb(doc, url);
 	});
 }
 
+
+function decodeEntities(str, doc) {
+	if (!str || !str.includes('&') || !doc.createElement) {
+		return str;
+	}
+	
+	var textarea = doc.createElement('textarea');
+	textarea.innerHTML = str;
+	return textarea.value;
```

Comment:
Did you get this from https://stackoverflow.com/a/7394787 ? If so, let's add a link, in case a future person wants to read up on it.

Reply:
Added!

Reply:
And yes, I did :)